learning/tour-of-beam/frontend/lib/models/user_progress.dart

```diff
@@ -22,6 +22,7 @@ part 'user_progress.g.dart';
 
 @JsonSerializable(createToJson: false)
 class UserProgressModel {
+  @JsonKey(name: 'id')
```

Comment:
Can we also rename this on the backend? `UserProgress.id -> unitId`
@eantyshev

Reply:
Or you can change the JSON field name here:

https://github.com/apache/beam/blob/37fb90c98d10b7d81dc195e83aafff14157fd8ed/learning/tour-of-beam/backend/internal/entity.go#L89

and just ask Evgeny to review.

Reply:
It's called `UnitProgress` on backend and `UserProgress` on frontend. Which one is the correct name? If it is `UnitProgress`, then `id` seems specific enough.

Reply:
Backend is always right because it is the origin of all entities. Always name things the way that backend does. Yes, `id` is no problem then.